UnitTests/GitCommands.Tests/Config/ConfigFileTest.cs

```diff
@@ -74,7 +74,7 @@ public void TestWithInvalidFileName()
                 File.WriteAllText(GetConfigFileName(), GetDefaultConfigFileContent(), GitModule.SystemEncoding);
             }
 
-            ConfigFile configFile = new(GetConfigFileName() + "\\", false);
+            ConfigFile configFile = new(GetConfigFileName() + "\\");
```

Comment:
Can/should we use the helper method "EnsureTrailingSlash" instead?

Reply:
> Can/should we use the helper method "EnsureTrailingSlash" instead?

This under full control of this test:
```
        private static string GetConfigFileName()
        {
            return Path.Combine(GetTempFolder(), "testconfigfile");
        }
```

Reply:
Ah, didn't pay attention that it was in the test